net-nds/phpldapadmin/phpldapadmin-1.2.6.6.ebuild

```diff
@@ -7,29 +7,39 @@ inherit webapp
 
 MY_PN="phpLDAPadmin"
 DESCRIPTION="phpLDAPadmin is a web-based tool for managing all aspects of your LDAP server"
-HOMEPAGE="http://phpldapadmin.sourceforge.net"
+HOMEPAGE="https://github.com/leenooks/phpLDAPadmin"
 SRC_URI="https://github.com/leenooks/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
-IUSE=""
 
 RDEPEND="
-	>=dev-lang/php-7.2[hash(+),ldap,session,xml,nls]
+	>=dev-lang/php-8.0[hash(+),ldap,session,xml,nls]
+	dev-libs/openssl:=
 	virtual/httpd-php
 "
+BDEPEND="
+	media-libs/libpng
+"
 S="${WORKDIR}/${MY_PN}-${PV}"
 
-# http://phpldapadmin.git.sourceforge.net/git/gitweb.cgi?p=phpldapadmin/phpldapadmin;a=commit;h=7dc8d57d6952fe681cb9e8818df7f103220457bd
 PATCHES=(
 	"${FILESDIR}/${PN}-1.2.1.1-fix-magic-quotes.patch"
+	"${FILESDIR}/${PN}-1.2.6.2-r1-default-templates.patch"
+	"${FILESDIR}/${PN}-1.2.6.4-getDN-htmlspecialchars.patch"
 )
 
 need_httpd_cgi
 
 src_prepare() {
-	mv config/config.php.example config/config.php
+	has_version "dev-libs/openssl:0/1.1" && \
+		eapply "${FILESDIR}/${PN}-1.2.6.5-openssl-1.patch"
```

Comment:
Not a fan of conditional patching, but since openssl-1.1 is on its way out I think we can keep this here for a few months.

But why is this needed?  By reading the linked issue, it seems they fixed it with aes-256-gcm and the fix is included in 1.2.6.6.  Can you test if 1.2.6.6 works with openssl 1.1 without the patch?